sashimi-webapp/index.html

```diff
@@ -28,6 +28,23 @@
       rel="stylesheet">
   </head>
   <body>
+    <!-- js-sequence-diagram does not support markdown-it, hence we are doing this -->
+    <script src="./vendors/dependencies/webfont.js"></script>
+    <script src="./vendors/dependencies/raphael.min.js"></script>
+    <script src="./vendors/dependencies/snap.svg-min.js"></script>
+    <script src="./vendors/dependencies/underscore-min.js"></script>
+    <script src="./vendors/sequence-diagram.js"></script>
+    <!-- flowchart.js does not support markdown-it, hence we are doing this -->
+    <script src="./vendors/flowchart-latest.js"></script>
+    <!-- Author of viz.js does not want to fix this issue, hence we are doing this -->
+    <!-- Author mention to use this unconventional/bad way of creating fake fs -->
+    <script>window.fs = {};</script>
+    <script src="./vendors/viz.js"></script>
+    <!-- mermaid does not support markdown-it, hence we are doing this -->
+    <script src="./vendors/mermaidAPI.js"></script>
```

Comment:
Changed in PR #375! Viz.js is already minified according to the author. But i have used viz-lite.js instead as we don't seem to be using the packages in viz.js that are removed from viz-lite.js. The rest are changed to their respective minified versions.